index.html

```diff
@@ -13,6 +13,8 @@
           <a href="#origin"><li>Origin Story</li></a>
           <a href="#story"><li>Earlier Years</li></a>
           <a href="#present"><li>Present </li></a>
+        <a href="game.html"><li>  <button type="button">Click Me!</button></li></a>
+    
         </ul>
       </nav>
         <section>
```

Comment:
this section is a sibling of the nav, so it should be indented to the same level